shared/src/main/scala-2.13/scala/util/parsing/input/ScalaVersionSpecificPagedSeq.scala

```diff
@@ -0,0 +1,7 @@
+package scala.util.parsing.input
+
+private[input] trait ScalaVersionSpecificPagedSeq[T] { self: PagedSeq[T] =>
+  // Members declared in scala.collection.Seq
+  override def iterableFactory: collection.SeqFactory[collection.IndexedSeq] = collection.IndexedSeq
```

Comment:
@julienrf why is this required?

Reply:
Maybe this is not anymore required, actually. Because we override it with the same body as the inherited one…